docs/architecture-decision-records/0019-remove-encryption-from-notification-topic.md

```diff
@@ -0,0 +1,42 @@
+# 19. Remove encryption from notifications SNS topic
+
+**Date:** 2024-06-28
+
+## Context
+
+We originally implemented KMS (Key Management Service) encryption on the notifications SNS (Simple Notification Service) topic to ensure tighter control over who could read the messages and to encrypt any potentially sensitive data at rest. The encrypted topic was intended to provide an additional layer of security for sensitive information.

```

Comment:
Should there be an apostrophe after "n" in "notifications" in "on the notifications"?